local-env-setup.sh

```diff
@@ -85,6 +85,22 @@ elif [ "$kernelname" = "Darwin" ]; then
         echo "Installing openjdk@8"
         brew install openjdk@8
     fi
+    for ver in 3.7 3.8 3.9; do
+      if brew ls --versions python@$ver > /dev/null; then
+          echo "python@$ver already installed. Skipping"
+          brew info python@$ver
+      else
+          echo "Installing python@$ver"
+          brew install python@$ver
+      fi
+      if [ ! $(type -P python$ver) > /dev/null 2>&1 ]; then
+        # For some python packages, brew does not add symlinks...
+        # TODO: use pyenv.
```

Comment:
What exactly is the TODO here? Will pyenv make symlinks for us?

Reply:
Reworded to: # TODO: Consider using pyenv to manage multiple installations of Python.